kvbc/tools/sparse_merkle_db/include/sparse_merkle_db_editor.hpp

```diff
@@ -185,14 +185,37 @@ struct GetValue {
   }
 };
 
+struct RemoveMetadata {
+  std::string description() const {
+    return "removeMetadata\n"
+           "Removes metadata and state transfer data from RocksDB";
+  }
+  std::string execute(const v2MerkleTree::DBAdapter &db_adapter, const CommandArguments &) const {
```

Comment:
Nit: be consistent in the use of function parameter names in the declaration.

Reply:
Done.